view/theme/frio/templates/admin/site.tpl

```diff
@@ -351,7 +351,9 @@
 			</h4>
 		</div>
 		<div id="admin-settings-relocate-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="admin-settings-relocate">
-
+			<div class="alert alert-danger">
+				<b>WARNING!</b> {{$relocate_warning}}
```

Comment:
Please move the `<b>WARNING! </b>` string to the translation label.

Reply:
You should not use the `<b>` tag to emphasise something, semantically the `<strong>` tag should be used.

Reply:
I'll have to create a new PR for this, right ?

Reply:
Yes, even though you can use the same branch.

Reply:
The sad thing is, that in most environments there is no visual difference between the emphasis `em` and strong emphasis `strong` to the italic `i` and bold `b` settings. Thanks for changing it!